roles/api-environment/tasks/express-api.yml

```diff
@@ -1,4 +1,20 @@
 ---
+  - name: "create docker script directory"
+    file:
+      path: "{{ express_api_docker_path }}"
+      state: directory
+      owner: root
+      group: root
+
+  - name: copy express-api run dockerfile to docker script directory
```

Comment:
It's not necessary but looks like the name of the rest tasks are quotes, can you also quote it for the code consistency?